File: lista_01/07/NumerosDivisiveisPor5.java
public class NumerosDivisiveisPor5 {
    public static void main(String[] args) {
        System.out.println("Números entre 1000 e 2000 que ao serem divisíveis por 5 produzem resto 3:");

        for (int i = 1000; i <= 2000; i++) {
            if (i % 5 == 3) {
                System.out.println(i);
            }
        }
    }
}

File: lista_01/07/NumerosDivisiveisPor5.py
def main():
    print("Números entre 1000 e 2000 que ao serem divisíveis por 5 produzem resto 3:")
    for i in range(1000, 2001):
        if i % 5 == 3:
            print(i)

if __name__ == "__main__":
    main()
